### DISCUSSION TITLE
More config options for spam filter

### DISCUSSION THREAD

**Discussion by AcaiiTV** [GriefPrevention/GriefPrevention#659] [Ideas]
As a server owner, I want to change certain parameters of the spam filter for in-game chat because I believe the default is too intense.

For example: 
-number of messages before auto-mute
-enable all caps

**Comment by sempervali** [comment]
Absolutely. I would rather be able to disable the caps protection alone instead of the entirety of the spam protection.
How do we bump this up? This is an old suggestion...
